README.md

```diff
@@ -1,33 +1,189 @@
+Grant a group access to a namespace with the specifed role
+(admin, edit, or view).
 
-An empty [`.aicoe-ci.yaml`](.aicoe-ci.yaml) is created here, disabling all checks via this CI provider by default. Documentation can be found [here](https://github.com/AICoE/aicoe-ci/).
+Usage:
+  opfcli grant-access namespace group role [flags]
 
-## Prow CI
+Flags:
+  -h, --help   help for grant-access
+```
 
-Prow is a CI provider developed for Kubernetes needs. Provides chat-ops management of pull requests, issues and declarative management for labels, branches and many more.
+Use "opfcli [command] --help" for more information about a command.
 
-We host our own deployment of Prow in Operate First available at [https://prow.operate-first.cloud/](https://prow.operate-first.cloud/).
+## Configuration
 
-Supported commands are listed [here](https://prow.operate-first.cloud/command-help). We have also enabled Prow to consume on-repository configuration files. You can specify your config in [`.prow.yaml`](.prow.yaml). Additional centralized configuration can be found in the [thoth-application repository](https://github.com/thoth-station/thoth-application/tree/master/prow/overlays/cnv-prod).
+The `opfcli` command will look for a configuration file `.opfcli.yaml`
+in two places:
 
-## Pre-commit
+- It first checks in the top level of the current git repository. If
+  you are running the `opfcli` command outside of a git repository it
+  will instead check the current directory.
+
+- If it doesn't find a local configuration file, it will look for
+  `~/.opfcli.yaml`.
+
+### Available configuration options
+
+- `app-name` -- sets the name of the directory containing your YAML
+  resources. This defaults to `cluster-scope`.
+
+## Examples
+
+### Create a project
+
+```
+opfcli create-project project1 group1 -d "This is project1"
+```
+
+This will result in:
+
+```
+cluster-scope/
+├── base
+│   ├── core
+│   │   └── namespaces
+│   │       └── project1
+│   │           ├── kustomization.yaml
+│   │           └── namespace.yaml
+│   └── user.openshift.io
+│       └── groups
+│           └── group1
+│               ├── group.yaml
+│               └── kustomization.yaml
+└── components
+    └── project-admin-rolebindings
+        └── group1
+            ├── kustomization.yaml
+            └── rbac.yaml
+```
+
+### Create a group
+
+```
+opfcli create-group group2
+```
+
+This will result in:
+
+```
+cluster-scope/
+└── base
+    └── user.openshift.io
+        └── groups
+            └── group1
+                ├── group.yaml
+                └── kustomization.yaml
+```
+
+### Grant access to a project
+
+```
+opfcli grant-access project1 group2 view
+```
+
+This will result in:
+
+```
+cluster-scope/components/project-view-rolebindings/
+└── group2
+    ├── kustomization.yaml
+    └── rbac.yaml
+```
+
+(And will modify
+`cluster-scope/base/core/namespaces/project1/kustomization.yaml`)
+
+## License
```

Comment:
There's already an GPL license besides readme (based on [ADR 0001](https://www.operate-first.cloud/blueprints/blueprint/docs/adr/0001-use-gpl3-as-license.md)). Do you want to claim copyright on this tool?

Reply:
According to the GPL:

> This involves adding two elements to each source file of your program: a copyright notice (such as “Copyright 1999 Terry Jones”), and a statement of copying permission, saying that the program is distributed under the terms of the GNU General Public License (or the Lesser GPL, or the Affero GPL).

While I haven't gotten around to including the license in every source file yet, I think we do need to include the copyright notice and license information *somewhere*, and the README seems like a good choice.

Reply:
We can change it to be "Copyright ... The Operate First Team", but in order to use the GPL, we need to (a) assert copyright and (b) say that we're using it.